<launch>
	<!-- main -->
	<node pkg="pc_projection_to_image" type="pc_projection_to_image_offline" name="pc_projection_to_image_offline" output="screen">
		<param name="load_rosbag_path" type="string" value="$(env HOME)/rosbag/sq2_velodyne/dkan_outdoor/2018-10-31-08-37-11_converted_transformed.bag"/>
		<param name="camera_info_name" type="string" value="/camera/color/camera_info"/>
		<param name="load_image_name" type="string" value="/camera/color/image_raw/compressed"/>
		<param name="save_image_name" type="string" value="/test"/>
		<!-- <param name="debug_hz" type="double" value="100"/> -->
		<param name="pc_0" type="string" value="/velodyne_points/transformed"/>
	</node>

	<!-- rviz -->
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find pc_projection_to_image)/rviz_config/offline_velodyne.rviz"/>
</launch>
